build.sbt

```diff
@@ -1,12 +1,14 @@
+organization := "gatling-mqtt"
```

Comment:
so shall i rework my pull request or will you clean these two issues while merging?